.eslintrc.yml

```diff
@@ -12,4 +12,5 @@ plugins:
 rules:

```

Comment:
also use the new eslint config. it's includes chai and mocha lints

Reply:
Done